solutions/remotemonitoring/scripts/all-in-one.yaml

```diff
@@ -958,7 +958,7 @@ spec:
             configMapKeyRef:
               name: deployment-configmap
               key: diagnostics.appinsights.instrumentationkey
-        - name: PCS_CONFIG_WEBSERVICE_URL
+        - name: PCS_CONFIG_URL
```

Comment:
I would not make changes in the RM but rather change the code in Diagnostics service to PCS_CONFIG_WEBSERVICE_URL. This is what is being used everywhere and will break the style that is being used even in Simulation except for Config url.